### PR TITLE
Home page redesign and general improvements

### PR DESCRIPTION
**Pull request by AbleOne**
1) Home page is updated according to new mockup
2) Favicon code updated to most modern including tab color-schemes
3) Main menu active states added
4) SCSS refactored and restructured
5) Updated npm modules
6) JS refactored and optimized

### CODE REVIEW

File: themes/bootique-theme/static/images/favicon/browserconfig.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 <browserconfig>
-  <msapplication>
-    <tile>
-      <square150x150logo src="/images/favicon/mstile.png"/>
-      <TileColor>#ffffff</TileColor>
-    </tile>
-  </msapplication>
+    <msapplication>
+        <tile>
+            <square150x150logo src="images/favicon/mstile-150x150.png"/>
```
Comment (IRus) [inline themes/bootique-theme/static/images/favicon/browserconfig.xml]: Are you sure that there are should be relative path?

File: themes/bootique-theme/static/images/favicon/site.webmanifest [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "",
+    "short_name": "",
+    "icons": [
+        {
+            "src": "images/favicon/android-chrome-192x192.png",
```
Comment (IRus) [inline themes/bootique-theme/static/images/favicon/site.webmanifest:6]: absolute path?

File: themes/bootique-theme/static/images/favicon/site.webmanifest [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+            "sizes": "192x192",
+            "type": "image/png"
+        },
+        {
+            "src": "images/favicon/android-chrome-512x512.png",
```
Comment (IRus) [inline themes/bootique-theme/static/images/favicon/site.webmanifest:11]: absolute path?

File: app/scripts/app.slickCarousel.ts [hunk 4]
```diff
@@ -0,0 +1,40 @@
+// Import slick-carousel from node_modules 
+// Loading from /scripts because https://github.com/kenwheeler/slick/pull/3307
+import "slick-carousel";
+
+// Testimonialz Slick slider
+export function testimonialsSlider() {
+	const slickSlider = $('.slider-center');
```
Comment (IRus) [inline app/scripts/app.slickCarousel.ts]: `import "slick-carousel";`

but 

` $('.slider-center');`

Please use `'` and `"` consistently. In JS `"` used most of the time.

File: app/styles/_docbook.scss [hunk 5]
```diff
@@ -1,293 +1,333 @@
...
+			>.h1 {
+				margin-top: map-get($spacers, 6);
+				margin-bottom: map-get($spacers, 4);
+			}
+
+			h2 {
+				margin-top: 0;
+			}
+
+			// [data-spy="scroll"] & {
```
Comment (IRus) [inline app/styles/_docbook.scss]: New commented code?

File: app/styles/_nav.scss [hunk 6]
```diff
@@ -0,0 +1,316 @@
...
+
+}
+
+.pure-menu-link,
+.pure-menu-disabled,
+.pure-menu-heading {
+  padding: 1em;
+}
+
+// .pull-right {
```
Comment (IRus) [inline app/styles/_nav.scss]: New commented code?

File: app/styles/_typography.scss [hunk 7]
```diff
@@ -1,130 +1,100 @@
...
+			color: inherit;
+
+			&:hover {
+				text-decoration: none;
+			}
+		}
+	}
+
+
+	// h1 {
```
Comment (IRus) [inline app/styles/_typography.scss]: Commented code

File: app/scripts/app.mediumPosts.ts [hunk 8]
```diff
@@ -0,0 +1,78 @@
+// Formating posts date
```
Comment (IRus) [inline app/scripts/app.mediumPosts.ts:1]: File uses tabs. Default in project - spaces. Please enable editorconfig plugin in sublime.

File: app/styles/_nav.scss [hunk 9]
```diff
@@ -0,0 +1,316 @@
...
+	      		transition-delay: .2s;
+	        
+	        // PROPERTIES TO ANIMATE: ENDPOINT
+	      		opacity: 1;
+	      }
+	    }
+	}  
+}
+
+// xs
```
Comment (IRus) [inline app/styles/_nav.scss:100]: Commented code

File: app/styles/_typography.scss (hunk 7 above)
Comment (IRus) [inline app/styles/_typography.scss]: Commented code.

File: app/styles/mixins/_custom-font-face.scss [hunk 10]
```diff
@@ -0,0 +1,46 @@
+/*
```
Comment (IRus) [inline app/styles/mixins/_custom-font-face.scss]: Licence?

**Review by IRus — CHANGES_REQUESTED** [review]
Overall codestyle issues:
1. Use `"` in javascript;
1. Use spaces instead of tabs;
1. Delete commented code;
1. Enable editorconfig.

File: app/styles/mixins/_custom-font-face.scss (hunk 10 above)
Comment (AbleOne) [reply app/styles/mixins/_custom-font-face.scss]: removed, strange thing - it is just scss mixin

File: app/scripts/app.mediumPosts.ts (hunk 8 above)
Comment (AbleOne) [reply app/scripts/app.mediumPosts.ts:1]: done

File: app/styles/_nav.scss (hunk 9 above)
Comment (AbleOne) [reply app/styles/_nav.scss:100]: This could be helpful for next releases. Commented it for now

File: app/styles/_typography.scss (hunk 7 above)
Comment (AbleOne) [reply app/styles/_typography.scss]: I'll finish with typography in next release when table of contents for docs will be styled and set up.
But I cleaned this up for now.

**Comment by AbleOne** [comment]
pushed changes
thanks for your revising!

**Review by IRus — APPROVED** [review]

**Comment by IRus** [comment]
Closed in favor of #13

**Pull request closed without merging**